src/pages/summer/Login/Login.scss

```
@@ -0,0 +1,138 @@
+@import '../../../styles/common.scss';
+
+.Login-summer main {
+  @include display-flex;
+  height: 742px;
+  margin: 14px 0 68px;
+
+  .right-container {
+    margin-left: -10px;
+
+    .login-container {
+      position: relative;
+      padding: 31px 40px 23px 40px;
+      border: 1px solid $color-lightgray;
+
+      h1 {
+        @include display-flex;
+        font-size: 43px;
+      }
+
+      .login-facebook {
+        position: relative;
+
+        &:before {
+          content: '';
+          position: absolute;
+          top: -33px;
+          left: 0;
+          width: 100%;
+          border-top: 1px solid $color-lightgray;
+        }
+
+        span:first-of-type {
+          position: absolute;
+          top: -40px;
+          left: 50%;
+          transform: translateX(-50%);
+          padding: 0 20px;
+          background: white;
+          @include bold-text(13px, $color-darkgray);
+        }
+
+        span:last-of-type {
+          @include display-flex(center, unset);
+
+          a {
+            padding: 4px 0 4px 24px;
+            @include bold-text(14px, $color-facebook);
+
+            &::before {
+              content: '';
+              position: absolute;
+              top: 3px;
+              left: 56px;
+              width: 16px;
+              height: 16px;
+              background-image: url('http://localhost:3000/images/summer/facebook.png');
+              background-size: cover;
+            }
+          }
+        }
+      }
+
+      .find-pw {
+        @include display-flex(center, unset);
+        margin-top: 18px;
+
+        a {
+          font-size: 12px;
+          color: $color-facebook;
+        }
+      }
+    }
+
+    .join-container {
+      @include display-flex(center, unset);
+      padding: 20px 0;
+      margin-top: 10px;
+      border: 1px solid $color-lightgray;
+
+      .joinning {
+        span {
+          margin-right: 2px;
+          font-size: 13px;
+          letter-spacing: 1px;
+        }
+        button {
+          @include bold-text(14px, $color-active);
+          cursor: pointer;
+        }
+      }
+    }
+
+    .app-container {
+      padding: 21px 36px 0;
+
+      & > p {
+        margin-bottom: 25px;
+        text-align: center;
+        font-size: 14px;
+      }
+      .app-btns {
+        @include display-flex(space-between, unset);
+        width: 280px;
+
+        button {
+          width: 135px;
+          font-size: 0;
+
+          img {
+            width: 100%;
+          }
+        }
+      }
+    }
+  }
+
+  /* media query */
+
+  /* 874px 이하 */
+  @media screen and (max-width: 874px) {
+    .mobile-display {
+      display: none;
+    }
+  }
+
+  /* 549px 이하 */
+  @media screen and (max-width: 549px) {
+    footer {
+      .footer-top {
+        padding: 0 20px;
+      }
+      .footer-top li {
+        padding: 0 2px 10px;
+      }
+    }
```

```suggestion
  @media screen and (max-width: 549px) {
    .footer-top {
      padding: 0 20px;
      
      li {
        padding: 0 2px 10px;
      }
    }
```
불필요한 nesting은 줄이고 중복되는 셀렉터는 합쳐 표현해줄 수 있을 것 같습니다